### ISSUE TITLE
Магус отключакус боргикус навечнус

### ISSUE THREAD

**Issue by AzzyDreemurr13** [TauCetiStation/TauCetiClassic#11642]
#### Подробное описание проблемы
В раунде в конце маг юзает спрятать борга в карман вместе с гсб, что по итогу на всегда выбивает борга из игры и без педальной магии никак

#### Что должно было произойти
Магус прятус боргус после когда отпуская борг приходит в себя

#### Что произошло на самом деле
Борг уснул вечным сном и без педали никак

#### Как повторить
Магус + боргус, запихнуть боргуса, и после отпустить, тада, боргусь полностью выведен из строя навсегда

#### Дополнительная информация:
Issue reported from  Round ID: 60000 (Tau Ceti Classic (RU))
Testmerges: ```#10434 #11272 #11464 #11571 #11597 #11604 ```
Reporting client version: 514.1589

**Comment by SiveanBen** [comment]
Не баг а фича

**Comment by AzzyDreemurr13** [comment]
Нет. Это не может быть фичей, что даже в роботехнике ничего не могут сделать и перезапустить борга.
